integration/gradle.lockfile

```diff
@@ -203,7 +203,7 @@ org.junit:junit-bom:5.7.0=compileClasspath,runtimeClasspath,testCompileClasspath
 org.junit:junit-bom:5.8.2=spotbugs
 org.jvnet.mimepull:mimepull:1.9.11=compileClasspath,testCompileClasspath
 org.jvnet.mimepull:mimepull:1.9.13=runtimeClasspath,testRuntimeClasspath
-org.liquibase:liquibase-core:4.2.1=runtimeClasspath,testRuntimeClasspath
+org.liquibase:liquibase-core:4.8.0=runtimeClasspath,testRuntimeClasspath
```

Comment:
this is a decent version bump... I've seen breaking changes sneak in with this sort of jump... Is anything dramatic listed on liquibase site/release-docs?

Reply:
I read through everything and provided links to the major release in the ticket comment-- https://broadworkbench.atlassian.net/browse/WOR-441?focusedCommentId=64825

There were no mention of breaking changes in any of the releases. Note that liquibase is used for database migrations-- I am hoping that all the existing migrations are applied in some part of the test or build process, but I don't actually know that.

Reply:
I double checked, and I didn't see anything that looked like a breaking change either, unless we were referencing externally defined entities with the xml parser.